intellij-plugin-structure/structure-fleet/src/main/kotlin/com/jetbrains/plugin/structure/fleet/FleetPluginManager.kt

```diff
@@ -116,4 +120,13 @@ class FleetPluginManager private constructor(private val extractDirectory: Path)
       return PluginCreationFail(UnableToReadDescriptor(DESCRIPTOR_NAME, "Bad descriptor format. Descriptor text: $serializedDescriptor" + "\n" + e.localizedMessage))
     }
   }
+
+  private fun parseThirdPartyDependenciesByPath(path: Path): List<ThirdPartyDependency> {
+    return if (path.exists()) {
```

Comment:
runCatching {}

Reply:
I would like not to ignore the problems here